src/article/shared/InlineGraphicComponent.js

```diff
@@ -0,0 +1,19 @@
+import { NodeComponent } from '../../kit'
```

Comment:
I think that the implementation of GraphicComponent.js and InlineGraphicComponent.js should be assimilated.

Reply:
Agree, but first we need to think what to do with displaying errors. 
E.g. when there is a file missing for a figure we are displaying a big red warning, but in case of a small inline graphic... I do not know.

Reply:
Right. @michael pls